todo-api/package.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Shouldn't `json-server` be a dependency of this particular mini-app? May just need to run a `yarn add json-server` from the `todo-api` directory.